addons/better-img-uploads/userscript.js

```diff
@@ -25,7 +25,7 @@ export default async function ({ addon, console, msg }) {
     });
     button.append(img);
     const input = Object.assign(document.createElement("input"), {
-      accept: ".svg, .png, .bmp, .jpg, .jpeg",
+      accept: ".svg, .png, .bmp, .jpg, .jpeg, .sprite3",
```

Comment:
https://github.com/scratchfoundation/scratch-gui/blob/a5bbc4c488aa530fa89cb24a0eecc07cd40f5d57/src/components/sprite-selector/sprite-selector.jsx#L123

Reply:
We should add `.sprite2` here